controllers/api/workoutRoutes.js

```diff
@@ -110,4 +110,26 @@ router.delete('/:id', withAuth, async (req, res) => {
     }
 });
 
+// Add Workout to users workouts database
```

Comment:
These details are only for the user table,
userworkouts is only user_id, workout_id and reps